prisma/schema.prisma

```diff
@@ -0,0 +1,25 @@
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "mongodb"
+  url      = env("DATABASE_URL")
+}
+
+model Room {
+  id        String  @id @map("_id")
+  title     String
+  isPrivate Boolean
+  password  String?
+  Users      User[]
+  hash String
+}
+
+model User {
+  id     String @id @map("_id")
+  name   String? @unique()
```

Comment:
the name should not be unique, it's unique by room, but we cannot create this rule on database.

and i would rename this field to "nickname"